spec/controllers/api/v1/classifications_controller_spec.rb

```diff
@@ -152,6 +152,24 @@ def create_gold_standard
       expect(json_response['classifications'].map{|c| c['id'].to_i})
         .to match_array(classifications.map(&:id))
     end
+
+    context "filtered by last_id" do
+      let!(:first) { create(:classification, user: authorized_user, project: project) }
+      let!(:second) { create(:classification, user: authorized_user, project: project) }
+
+      it "does not include classifications prior to given id", :aggregate_failures do
+        get :project, last_id: first.id, project_id: project.id
+        resources = json_response[api_resource_name]
+        expect(resources.length).to eq 1
+        expect(resources).not_to include Classification.first
+      end
+
+      it 'returns nothing if project id' do
+        get :project, last_id: first.id
+        resources = json_response[api_resource_name]
+        expect(resources.length).to eq 0
```

Comment:
final piece is to test the error message is correctly serialized, see [here](https://github.com/zooniverse/Panoptes/blob/219854367f346a0376906cf4389e078f57a6a80c/app/controllers/api/api_controller.rb#L35) for the api controller error handling for good error messages.